src/e3/anod/loader.py

```diff
@@ -69,7 +69,21 @@ def __init__(
         if not os.path.isdir(spec_dir):
             raise SandBoxError(f"spec directory {spec_dir} does not exist")
         self.spec_dir = spec_dir
-        self.api_version = __version__
+
+        # Read read the API version file
+        version_file = os.path.join(self.spec_dir, "VERSION")
+        if os.path.isfile(version_file):
+            with open(version_file) as f:
+                content = f.read().strip()
+                if ":" not in content:
+                    raise SandBoxError(
+                        f"invalid VERSION file in spec dir {self.spec_dir}"
+                    )
+                self.api_version = content.split(":")[1].strip()
```

Comment:
We may check that the API version retrieved this way belongs to the `SUPPORTED_API` tuple ?

Reply:
thanks !